src/synthesizer/line.py

```diff
@@ -9,30 +9,130 @@
 from synthesizer.conversions import lnu_to_llam
 from synthesizer.units import Quantity
 from synthesizer import exceptions
+from unyt import Angstrom
 
 
 def get_line_id(id):
     """
     A class representing a spectral line or set of lines (e.g. a doublet)
 
     Arguments
-    ----------
-    id : str, list, tuple
-        a str, list, or tuple containing the id(s) of the lines
+        id (str, list, tuple)
+            a str, list, or tuple containing the id(s) of the lines
 
     Returns
-    -------
-    string
-        string representation of the id
-
-    """ "H 1 6564.62A"
+        id (str)
+            string representation of the id
+    """
 
     if isinstance(id, list):
         return ",".join(id)
     else:
         return id
 
 
+# dictionary of common line labels to use by default
+special_line_labels = {
```

Comment:
This should not be a global. Can it be included on a `LineCollection`?

Reply:
Doesn't make sense in LineCollection but I've moved it into the `get_line_label function` which is the only place it's used.